notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>I don't think "Clear" is the right word here.  Consider:</p><p>"Move hexapod to in-focus position"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
Thanks 👍